squeal-postgresql/src/Squeal/PostgreSQL/Session/Result.hs

```diff
@@ -138,6 +147,38 @@ nfields = liftResult LibPQ.nfields
 resultStatus :: MonadIO io => Result y -> io LibPQ.ExecStatus
 resultStatus = liftResult LibPQ.resultStatus
 
+{- |
+Returns the command status tag from the SQL command
+that generated the `Result`.
+Commonly this is just the name of the command,
+but it might include additional data such as the number of rows processed.
+-}
+cmdStatus :: MonadIO io => Result y -> io Text
+cmdStatus = liftResult (getCmdStatus <=< LibPQ.cmdStatus)
+  where
+    getCmdStatus = \case
+      Nothing -> throwIO $ ConnectionException "cmdStatus"
+      Just bytes -> return $ Text.decodeUtf8 bytes
+
+{- |
+Returns the number of rows affected by the SQL command.
+This function returns `Just` the number of
+rows affected by the SQL statement that generated the `Result`.
+This function can only be used following the execution of a
+SELECT, CREATE TABLE AS, INSERT, UPDATE, DELETE, MOVE, FETCH,
+or COPY statement,or an EXECUTE of a prepared query that
+contains an INSERT, UPDATE, or DELETE statement.
+If the command that generated the PGresult was anything else,
+`cmdTuples` returns `Nothing`.
+-}
+cmdTuples :: MonadIO io => Result y -> io (Maybe LibPQ.Row)
+cmdTuples = liftResult (getCmdTuples <=< LibPQ.cmdTuples)
+  where
+    getCmdTuples = \case
+      Nothing -> throwIO $ ConnectionException "cmdStatus"
```

Comment:
This should say cmdTuples, and I prefer the convention you have of `LibPQ.cmdTuples`

Reply:
Thanks, fixed.